sku.0/sys.server/compiled/game/script/systems/combat/combat_actions.java

```diff
@@ -10483,21 +10478,17 @@ public int bm_revive_pet_1(obj_id self, obj_id target, String params, float defa
             setCommandTimerValue(self, TIMER_COOLDOWN, 0.0f);
             return SCRIPT_OVERRIDE;
         }
-        if (getDistance(getLocation(self), getLocation(beast)) > 7.0f)

```

Comment:
This block should still be here, just set to > 85? Otherwise, if a user is farther away than the new maximum distance of 85m they won't get the feedback that they're too far away with the cancel/reset of the command? 

Make sure you also add an adjustment to `maxRangeToTarget` in the command table or this change won't work either way as it's set to 7 currently.

Reply:
Re added the out of range check below